lib/dfe_signin/user_access.rb

```diff
@@ -0,0 +1,40 @@
+module DfeSignin
+  class UserAccess
+    attr_accessor :organisation_id,
+                  :user_id,
+                  :body
+
+    def initialize(organisation_id:, user_id:)
+      self.organisation_id = organisation_id
+      self.user_id = user_id
+    end
+
+    def call
+      request = Net::HTTP::Get.new(uri)
+      request["Authorization"] = "bearer #{DfeSignin::Utils.generate_jwt_token}"
+      request["Content-Type"] = "application/json"
+
+      response = Net::HTTP.start(uri.hostname, uri.port, use_ssl: true) { |http|
+        http.request(request)
+      }
+
+      raise ExternalServerError if response.code.eql?("500")
```

Comment:
Might be helpful if we passed on the message returned in the response into the exception to give us more context about what went wrong. Assuming we get some body back with the response from sign-in.